inc/sprite-stat-ids.inc

```diff
@@ -0,0 +1,192 @@
+; Information obtained from https://docs.google.com/spreadsheets/d/1Tcem3usNfxDFJp_uoe79GgJyWZabJaLW8uKzQjwZsD8
```

Comment:
Nice source but is that info literally no where else found? Metconst, mage, anywhere?

Google sheet links are usually much more volatile so I would prefer to link to somewhere else instead